dashboard.php

```diff
@@ -17,8 +17,52 @@
             </ul>
         </nav>
     </header>
+    <?php 
+    require_once('php/connection.php');
+    session_start();
+    $dsn = DB_DSN;
+    $user = DB_USERNAME;
+    $pass = DB_PASSWORD;
+
+    $opt = [
+        PDO::ATTR_ERRMODE            => PDO::ERRMODE_EXCEPTION,
+        PDO::ATTR_DEFAULT_FETCH_MODE => PDO::FETCH_ASSOC,
+        PDO::ATTR_EMULATE_PREPARES   => false,
+    ];
+    $pdo = new PDO($dsn, $user, $pass, $opt);
```

Comment:
Everything from `$dsn = DB_DSN` to this line should be removed as it does nothing - it's all handled in the `newConn` function.